card.css

```diff
@@ -0,0 +1,178 @@
+.content-container {
+    background-color: black;
+}
+
+.main-card-container {
+    background-color: #FFFFF0;
+    position: relative;
+    height: fit-content;
+    margin-right: 10px;
+}
+
+.main-card-container:nth-child(1) {
+    margin-left: 10px !important;
+}
+
+.card-horizontal .card-container-horizontal {
+    display: flex;
+    flex-direction: row;
+    max-height: fit-content;
+}
+
+.image-badge-container {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+}
+
+.image-badge-container img {
+    width: 111px;
+}
+
+.text-container {
+    margin-left: 1rem;
+}
+
+.badge {
+    background-color: var(--primary-color);
+    font-family: "Roboto";
+    letter-spacing: 2px;
+    font-size: 15px;
+    padding: 5px;
+    position: absolute;
+    top: 0rem;
+    left: 0rem;
+}
+
+.sub-text-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.sub-title {
+    font-size: 19px;
+}
+
+.card-desc {
+    font-size: 20px;
+    margin-bottom: 1rem;

```

Comment:
You can have given "margin:1rem 0;" this would set the 1rem margin for top and bottom and 0 for left and right.